src/main/java/net/minecraftforge/items/wrapper/CombinedInvWrapper.java

```diff
@@ -30,6 +31,11 @@ public CombinedInvWrapper(IItemHandlerModifiable... itemHandler)
         this.slotCount = index;
     }
 
+    public CombinedInvWrapper(IItemHandlerModifiable... itemHandler)
```

Comment:
Might want a `TODO 1.20: remove to encourage broader support of IOGuarantees` ?

Reply:
Good point, I should probably deprecate it for removal